config/examples/Creality/CR-10 V2/Configuration.h

```diff
@@ -845,11 +845,13 @@
  * The probe replaces the Z-MIN endstop and is used for Z homing.
  * (Automatically enables USE_PROBE_FOR_Z_HOMING.)
  */
-#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
+//#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
 
 // Force the use of the probe for Z-axis homing
 //#define USE_PROBE_FOR_Z_HOMING
-
+#if ENABLED(CR10V2_BLTOUCH)
+  #define USE_PROBE_FOR_Z_HOMING
+#endif
```

Comment:
Won't you need `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` enabled if `CR10V2_BLTOUCH` is not defined?

Reply:
As far as I understand the purpose of `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN ` is to use the `Z_STOP_PIN` instead of the `Z_MIN_PROBE_PIN`. The original code define this and yet did a override of the Z_STOP_PIN with the pin number of probe pin on the Creality board which is redundant steps and yet with the side effect of disabling the endstop.

Reply:
You are right. I wasn’t thinking straight with my earlier comment! I was thinking it would be needed for an endstop but that is obviously now true.